Sources/Cells/TextFieldTableViewCell/TextFieldTableViewCell.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  Copyright © 2019 Rosberry. All rights reserved.
+//
+
+import UIKit
+
+final class TextFieldTableViewCell: BaseTableViewCell {
+
+    private var plugin: TextFieldPlugin?
+
+    // MARK: - Subviews
+
+    private lazy var textField: UITextField = {
+        let view = UITextField()
+        view.borderStyle = .roundedRect
+        view.delegate = self
+        view.addTarget(self, action: #selector(textFieldValueChanged), for: .editingChanged)
+        return view
+    }()
+
+    // MARK: - Lifecycle
+
+    override func setup() {
+        super.setup()
+        accessoryView = textField
+    }
+
+    override func configure(with plugin: Plugin) {
+        super.configure(with: plugin)
+        self.plugin = plugin as? TextFieldPlugin
+
+        if let plugin = self.plugin {
+            textField.placeholder = plugin.placeholder
+            textField.text = plugin.initialValue
+            textField.keyboardType = plugin.keyboardType
+        }
+    }
+
+    override func layoutSubviews() {
+        textField.bounds = CGRect(origin: .zero, size: .init(width: contentView.bounds.midX, height: 40))
```

Comment:
You should set text field's width base on bounds of the whole cell, not its `contentView`; otherwise it'll do some funny stuff during swipe (add / remove from favorites)

Reply:
@stason314, let's fix this one and merge it.